tests/scripts/scripts.csproj

```diff
@@ -18,7 +18,7 @@
       <Version>$(XunitPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.DotNet.BuildTools.TestSuite">
-      <Version>1.0.0-prerelease-00629-04</Version>
+      <Version>1.0.0-prerelease-00704-04</Version>
```

Comment:
Package v1.0.0-prerelease-00629-04 is not available, updated to version resolved by NuGet